src/python/pants/core/goals/test.py

```diff
@@ -454,6 +455,18 @@ def calculate_from_global_options(self, test: TestSubsystem) -> Optional[int]:
         return result
 
 
+class TestExtraEnvVarsField(StringSequenceField, metaclass=ABCMeta):
```

Comment:
Good idea. Maybe update other backends to use this too?

Reply:
yeah, I guess it's better to include those here than creating a new PR

Reply:
New internal PR would be fine too. Either way :)